sales_tax.rb

```diff
@@ -1,9 +1,74 @@
-class Product
+require_relative("print_receipt.rb")
+
+$exmpt = ["chocolates","book","pills","chocolate"]
```

Comment:
instead of global variable use constant here and also use `%w` syntax for array of strings.